samples/kubeflow-tf/kubeflow-training-classification.py

```diff
@@ -13,75 +13,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
 import kfp.dsl as dsl
 import kfp.gcp as gcp
-import datetime
-
-def dataflow_tf_transform_op(train_data: 'GcsUri', evaluation_data: 'GcsUri', schema: 'GcsUri[text/json]', project: 'GcpProject', preprocess_mode, preprocess_module: 'GcsUri[text/code/python]', transform_output: 'GcsUri[Directory]', step_name='preprocess'):
-    return dsl.ContainerOp(
-        name = step_name,
-        image = 'gcr.io/ml-pipeline/ml-pipeline-dataflow-tft:3b949b37aa2cefd3180398d59116f43ce965a2a6',
-        arguments = [
-            '--train', train_data,
-            '--eval', evaluation_data,
-            '--schema', schema,
-            '--project', project,
-            '--mode', preprocess_mode,
-            '--preprocessing-module', preprocess_module,
-            '--output', transform_output,
-        ],
-        file_outputs = {'transformed': '/output.txt'}
-    )
 
+from kfp.components import ComponentStore
 
-def kubeflow_tf_training_op(transformed_data_dir, schema: 'GcsUri[text/json]', learning_rate: float, hidden_layer_size: int, steps: int, target, preprocess_module: 'GcsUri[text/code/python]', training_output: 'GcsUri[Directory]', step_name='training', use_gpu=False):
-    kubeflow_tf_training_op = dsl.ContainerOp(
-        name = step_name,
-        image = 'gcr.io/ml-pipeline/ml-pipeline-kubeflow-tf-trainer:3b949b37aa2cefd3180398d59116f43ce965a2a6',
-        arguments = [
-            '--transformed-data-dir', transformed_data_dir,
-            '--schema', schema,
-            '--learning-rate', learning_rate,
-            '--hidden-layer-size', hidden_layer_size,
-            '--steps', steps,
-            '--target', target,
-            '--preprocessing-module', preprocess_module,
-            '--job-dir', training_output,
-        ],
-        file_outputs = {'train': '/output.txt'}
-    )
-    if use_gpu:
-        kubeflow_tf_training_op.image = 'gcr.io/ml-pipeline/ml-pipeline-kubeflow-tf-trainer-gpu:3b949b37aa2cefd3180398d59116f43ce965a2a6'
-        kubeflow_tf_training_op.set_gpu_limit(1)
-    
-    return kubeflow_tf_training_op
+cs = ComponentStore()
```

Comment:
I don't think we should expose it yet.

Reply:
Hmm. Why?
It's pretty simple and straightforward comprehension for the user.
It just tries multiple URLs when loading the component.
Here it's useful so that the sample does not break when it moves from branch to master.

Reply:
We can replace it with full URLs once the PR is in master (and the URLs become correct).